src/__init__.py

```diff
@@ -31,7 +31,7 @@ def initCSS(self, *args):
       transform: translateY(-0.2em);
       font-size: .6em;
       -webkit-text-stroke-width: initial;
-      color: #666;
+      color: inherit;
```

Comment:
Alternative would be to use this `.nightMode` class: https://docs.ankiweb.net/templates/styling.html#night-mode